pkg/controllers/ipaddressallocation/ipaddressallocation_controller.go

```diff
@@ -109,8 +105,14 @@ func (r *IPAddressAllocationReconciler) Reconcile(ctx context.Context, req ctrl.
 	log.Info("reconciling IPAddressAllocation CR", "IPAddressAllocation", req.NamespacedName)
 	metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerSyncTotal, MetricResType)
 	if err := r.Client.Get(ctx, req.NamespacedName, obj); err != nil {
-		log.Error(err, "unable to fetch IPAddressAllocation CR", "req", req.NamespacedName)
-		return resultNormal, client.IgnoreNotFound(err)
+		if client.IgnoreNotFound(err) == nil {
+			err = r.Service.DeleteIPAddressAllocationByNamespacedName(req.Namespace, req.Name)
+			if err != nil {
+				log.Error(err, "failed to delete IPAddressAllocation", "IPAddressAllocation", req.NamespacedName)
```

Comment:
Log strings should be capitalized?
```suggestion
				log.Error(err, "Failed to delete IPAddressAllocation", "IPAddressAllocation", req.NamespacedName)
```